src/components/UserRecentProofsDialog.vue

```diff
@@ -33,7 +38,8 @@ export default {
     return {
       userProofList: [],
       userProofTotal: null,
-      userProofPage: 1,
+      userProofPage: 0,
+      userProofType: this.$route.path.endsWith('/receipt') ? 'RECEIPT' : 'PRICE_TAG',
```

Comment:
the issue here is that the component's behavior will rely on the url. it would be better to be an optional parameter (props)

Reply:
I fixed it here :) https://github.com/openfoodfacts/open-prices-frontend/pull/589/commits/97f9852cb537417c95f65b6d1b5e8b70771381cc

Reply:
Nice, thanks! 👍 

However, if I'm understanding it correctly now that this filter is in place, the 'add a price' button on a specific proof id page with the `RECEIPT` type won't show the selected proof in the recent proofs dialog box on the add price page anymore. That could maybe be resolved by combining all the 'add price' pages into the add multiple page instead and add a proof type selector on the page to choose between price tag and receipt?